<ui version="4.0" >
 <author>Remy Rouvin</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>720</height>
   </rect>
  </property>
  <property name="mouseTracking" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>PockEmul</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_PockEmul" />
    <addaction name="separator" />
    <addaction name="actionCheck_for_Updates" />
   </widget>
   <widget class="QMenu" name="menuMonitor_2" >
    <property name="title" >
     <string>Monitor</string>
    </property>
    <addaction name="actionLog_Messages" />
    <addaction name="actionAnalogic_Port" />
   </widget>
   <widget class="QMenu" name="menuRoot" >
    <property name="title" >
     <string>Session</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="separator" />
    <addaction name="actionExit" />
    <addaction name="separator" />
    <addaction name="actionSkin_Editor" />
   </widget>
   <addaction name="menuRoot" />
   <addaction name="menuMonitor_2" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionAbout_PockEmul" >
   <property name="text" >
    <string>About PockEmul</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionOn_Off" >
   <property name="text" >
    <string>On/Off</string>
   </property>
  </action>
  <action name="actionSoft_Reset" >
   <property name="text" >
    <string>Soft Reset</string>
   </property>
  </action>
  <action name="actionHard_Reset" >
   <property name="text" >
    <string>Hard Reset</string>
   </property>
  </action>
  <action name="actionPrinter" >
   <property name="text" >
    <string>Printer...</string>
   </property>
  </action>
  <action name="actionSerial_Interface" >
   <property name="text" >
    <string>Serial Interface</string>
   </property>
  </action>
  <action name="actionLog_Messages" >
   <property name="text" >
    <string>Log Messages...</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Check for Updates...</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSkin_Editor" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Skin Editor ...</string>
   </property>
  </action>
  <action name="actionExtensions" >
   <property name="text" >
    <string>Extensions...</string>
   </property>
  </action>
  <action name="actionRegisters" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Registers...</string>
   </property>
  </action>
  <action name="actionI_O_Ports" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>I/O Ports...</string>
   </property>
  </action>
  <action name="actionDesassembler" >
   <property name="text" >
    <string>Desassembler</string>
   </property>
  </action>
  <action name="actionAnalogic_Port" >
   <property name="text" >
    <string>Analogic Port</string>
   </property>
  </action>
  <action name="actionDfgdf" >
   <property name="text" >
    <string>dfgdf</string>
   </property>
  </action>
  <action name="actionInternal_CPU_Memory" >
   <property name="text" >
    <string>Internal CPU Memory</string>
   </property>
  </action>
  <action name="actionExternal_Memory" >
   <property name="text" >
    <string>External Memory</string>
   </property>
  </action>
  <action name="actionSerial_Console" >
   <property name="text" >
    <string>Serial Console</string>
   </property>
  </action>
  <action name="actionTape" >
   <property name="text" >
    <string>Tape</string>
   </property>
  </action>
  <action name="actionKeyboard" >
   <property name="text" >
    <string>Keyboard</string>
   </property>
  </action>
  <action name="actioncpu100" >
   <property name="text" >
    <string>100%</string>
   </property>
  </action>
  <action name="actioncpu200" >
   <property name="text" >
    <string>200%</string>
   </property>
  </action>
  <action name="actioncpu300" >
   <property name="text" >
    <string>300%</string>
   </property>
  </action>
  <action name="actioncpu500" >
   <property name="text" >
    <string>500%</string>
   </property>
  </action>
  <action name="actioncpumax" >
   <property name="text" >
    <string>Maximum speed</string>
   </property>
  </action>
  <action name="actionNewLink" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionIdentify" >
   <property name="text" >
    <string>Identify</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>255</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
